src/Maker/DddEntityCommandMaker.php

```diff
@@ -37,7 +37,7 @@ protected function getRequiredOptions () : array
      */
     protected function getExtraVariables (InputInterface $input) : array
     {
-        $domain = $input->getOption('domain-name');
+        $domain = $input->getOption(parent::DOMAIN_NAME_OPTION_KEY);
         $entity = $input->getOption('entity-name');
         $actionName = new UnicodeString($input->getOption("command-action"));
         $actionName = $actionName->camel()->title()->toString();
```

Comment:
Unrelated (and invisible) to this PR:

I'd much rather prefer having a service that acts as a registry, which generates the naming strategies for various domain languages, so that it could be easily extended.

E.g. this code:

```suggestion
        $actionName = new UnicodeString($input->getOption("command-action"));
        $actionName = $actionName->camel()->title()->toString();

        if ("en" === $this->getDomainLanguage($domain)) {
            $commandNamespace = $actionName . $entity;
        }
        else {
            $commandNamespace = $entity . $actionName;
        }
```

would become this:

```suggestion
        $actionName = new UnicodeString($input->getOption("command-action"));
        
        $namingStrategy = $this->domainNamingStrategyRegistry->getForDomain($domain);
        $commandNamespace = $namingStategy->generateCommandNamespace(new DomaindNamingStategyInput($domain, $actionName, $entity));
```

Reply:
We'd probably need to spend a little bit more time refining those concepts, but in the end we could gain a lot of flexibility and even some project-specific features since the project could just expose some `DomainNamingStrategyService`s, which are automatically picked up and used by this bundle.

Reply:
Continued in #7